storage/kubernetes/types.go

```diff
@@ -167,6 +167,14 @@ type Client struct {
 
 	Name    string `json:"name,omitempty"`
 	LogoURL string `json:"logoURL,omitempty"`
+
+	SAMLInitiated SAMLInitiatedConfig `json:"samlInitiated"`
```

Comment:
❓ How about adding `,omitempty`? I suspect _most_ people won't use this right away... 🤔